projects/view/client/rtProject/rtProjectInfoSection.html

```diff
@@ -0,0 +1,45 @@
+<template name="rtProjectInfoSection">
```

Comment:
This is also about simplicity, and defining indentation rules for different file types may be confusing. Simply choosing one indentation rule makes it easy to remember and adhere to.